lib/widgets/home_page_article.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:news_app/domain/models/article.dart';
+import 'package:intl/intl.dart';
+
+class HomePageArticle extends StatelessWidget {
+  const HomePageArticle({
+    super.key,
+    required this.article,
+    required this.size,
+  });
+
+  final Article article;
+  final double size;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: EdgeInsets.all(size * 0.05),
```

Comment:
What is `0.05` value stands for? try to avoid using magic numbers.

Reply:
should i set that the value in a variable and use the variable instead so it can be more readable ?